### PR TITLE
hid some stuff, polished other stuff.

### PR DESCRIPTION
**Pull request by snack8**
AWWWW YISH YO. POLISH UP IN THE HIZZY FO SHIZZY

### CODE REVIEW

File: app/images/jumbotronBackgroundBlueLayer5.svg [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="303.59174mm"
```
Comment (halfspiral) [inline app/images/jumbotronBackgroundBlueLayer5.svg]: Should be saved as pixels

File: app/images/jumbotronBackgroundOrangeLayer5.svg [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="294.72064mm"
```
Comment (halfspiral) [inline app/images/jumbotronBackgroundOrangeLayer5.svg]: Should be saved as pixels

**Comment by snack8** [comment]
FIXED IT

**Comment by halfspiral** [comment]
+1

**Comment by halfspiral** [comment]
+:100:

**Pull request merged**